docs/blog/2020-04-17-gatsby-survey-goes-command-line/index.md

```diff
@@ -0,0 +1,103 @@
+
+![Pie chart with results requested by survey respondents](./improve_results.png "Chart showing responses to how Gatsby can improve survey question")
+
+Those who would be “very disappointed” without Gatsby also have high expectations for Gatsby’s improvement. The number one reported problem is unclear error messages, especially when it comes to GraphQL. The second most commonly mentioned issue is slow builds (no surprise and we’re working on it!). These survey results are confirmation that we should move fast on these things and devote more resources to improving/solving them.
+
+## How can you help improve Gatsby?
+
+We already knew that error messages needed to be improved (see [this RFC up for improving error messages](https://github.com/gatsbyjs/rfcs/pull/37)) and have been preparing usability testing for error messages. The number one way we can use help in improving Gatsby is for you to participate in error message testing!
+
+Here are two ways you can help by testing error messages:
+
+1. [Setup a usability test with us](https://calendly.com/shannon-soper/gatsby-usability). If you do this, don’t “cheat” on the test and look up the errors in the repo ahead of time! It’s most useful for us to see you face the error message for the first time, without preparation.
+2. [Clone this repo](https://github.com/gatsbyjs/error-testing) and find two sites under `error-testing/sites/`. Screen record yourself running one site at a time to test how fast you can resolve the errors in them. Send the screen recording to shannon@gatsbyjs.com!
+
+## How will the data be used?
+
+Anyone in the community, including employees, can read this and all future blog posts reporting results. This way we can publicly compare survey results side-by-side to measure the success of different features; for example, if one of our blog post summaries shows that most people have problems with unclear error messages, and the next post shows they no longer have that problem, all Gatsby community members will know something has changed.
+
+## Thank you and keep the feedback coming!
+
+Thank you again to all who have responded to the survey so far; your responses help guide the focus of the community to answer real-world needs. If you haven’t filled out the survey yet, you can run `gatsby feedback` at any time after upgrading your version of gatsby in a project’s `package.json` file, and then rerunning dependency and install commands.
+
+Otherwise, wait to see the survey appear in a CLI near you! The plan is to do this every 3 months, at the most. (Again, if you’d like to opt out, visit https://www.gatsbyjs.org/docs/CLI-feedback/#how-to-opt-out).
```

Comment:
```suggestion
Otherwise, wait to see the survey appear in a CLI near you! The plan is to do this every 3 months, at the most. (Again, if you’d like to opt-out, visit the [how to opt-out docs](/docs/CLI-feedback/#how-to-opt-out)).
```